### PR TITLE
Reactive Extensions Demo - examples/<user-list>

### PR DESCRIPTION
**Pull request by RobertChristopher**
Component with RxJS reactive data flow.

[Reactive Extensions](https://github.com/Reactive-Extensions/RxJS)

### CODE REVIEW

**Comment by RobertChristopher** [comment]
@snuggs All good to merge?

**Comment by snuggs** [comment]
Let's fix this bug first. #44 being fixed in #48

**Comment by snuggs** [comment]
@RobertChristopher also a convention is to remove WIP label when done.

**Comment by snuggs** [comment]
@brandondees a pretty solid example of using snuggsi with RxJS. I have never even seen it before @RobertChristopher showed me. I think this is a great example of using a bespoke data store as `this.context` within the custom element.

I'm going to make a few architectural changes before merging. /cc @mrbernnz @btakita @scottmacdowell

**Comment by snuggs** [comment]
@RobertChristopher rebased atop `master` branch.

File: examples/user-list/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<script defer>
+
+Element `user-list`
+
+(class extends Reactive ( HTMLElement ) {
+
+  onclick ()
+    { this.actions.onclick () }
+
+  onstatechange (state) {
```
Comment (snuggs) [inline examples/user-list/index.html]: Perhaps `onstatechange` should go in `reactive.js` Unless we want to provide this API to the end component author. Keeping this handler here doesn't feel very _"batteries included"_  As @brandondees would say.

**Pull request merged**